README.md

```diff
@@ -94,151 +88,143 @@ Whether or not this is the final proposed set of features is yet undecided.
-  by which rendering work can be avoided. User-agent features may modify this
-  attribute, causing on-demand rendering, if desired. The developer may listen
-  to this on-demand rendering via a MutationObserver and respond to it before
-  rendering occurs. `rendersubtree`, when present, forces `style` and `layout`
-  containment, plus `size` containment if invisible. This ensures minimal
-  invalidation of the rest of the document when rendering occurs.
-
-* A `content-size` attribute (early draft spec
-  [here](http://tabatkins.github.io/specs/css-content-size/) that specifies how
-  much space a subtree should take up. This is intended to allocate a
-  placeholder size for content marked as invisible by `rendersubtree` (since it
-  already has `size` containment, as mentioned above). (*Please note that this
-  property in particular has been discussed and changed quite a bit; the latest
-  discussion can be found [here](https://github.com/w3c/csswg-drafts/issues/4531)*)
-
-* An `updateRendering` method on Element objects. This can be used to pre-render
-  content within a subtree marked with `rendersubtree` as invisible to make it
-  ready for display or measurement.
+1. A new `render-subtree` CSS property (spec draft to be added soon).
+  This property controls whether DOM subtrees affected by the property do or do
+  not render, and is the mechanism by which rendering work can be avoided. Some
+  values of `render-subtree` allow the user-agent to automatically manage
+  whether subtrees affected are rendered or not. Other values give the developer
+  complete control of subtree rendering. Note that the names of the tokens are
+  being [discussed](https://github.com/WICG/display-locking/issues/110).
+  However, the brief description of the tokens is below:
+    * `render-subtree: invisible`: this configuration allows the user-agent to
+      automatically manage whether content is rendered or not.
+    * `render-subtree: invisible skip-activation`: this configuration gives the
+      developer complete control of when the subtree is rendered. Neither the
+      user-agent nor its features should need to process or render the subtree.
+    * `render-subtree: invisible skip-viewport-activation`: this configuration
+      allows the developer to control rendering, but it allows user-agent
+      features such as find-in-page to process the subtrees and fire the
+      activation event (described below).
+
+  It is also worth noting that when the element is not rendered, then
+  `contain: layout style size;` is added to its style to ensure that the
+  subtree content does not affect elements outside of the subtree.
+  Furthermore, when the element is rendered in the `render-subtree: invisible`
+  configuration (i.e. the user-agent decides to render the element), then
+  `contain: layout style;` applies to the element.
+
+2. A new `contain-intrinsic-size` CSS property (spec draf to be added soon). This
```

Comment:
I'm late, but I noticed a typo (draf -> draft)

Reply:
This was fixed in a separate commit.